<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>510</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>510</width>
    <height>200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ableton Sans Light</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="mainSettingsFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="TabWidget" name="tabWidget">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::West</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="usesScrollButtons">
         <bool>true</bool>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="interfaceTab">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <attribute name="title">
          <string>Look</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="settings1_frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <property name="horizontalSpacing">
              <number>86</number>
             </property>
             <property name="leftMargin">
              <number>45</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="Theme_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Ableton Sans Light</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Theme background</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="ComboBox" name="ThemeComboBox_Bckgrnd"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Ableton Sans Light</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Theme color</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="ComboBox" name="ThemeComboBox_Color"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="axis_size_label">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>15</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Ableton Sans Light</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Axis font-size</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="SpinBox" name="axis_font_size_spinBox">
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="axis_label_font_size">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>20</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Ableton Sans Light</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Axis label font-size</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="SpinBox" name="axis_label_font_size_spinBox">
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="value">
                <number>14</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Page2hidden">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <attribute name="title">
          <string/>
         </attribute>
        </widget>
        <widget class="QWidget" name="FileTab">
         <attribute name="title">
          <string>File</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QFrame" name="settings3_frame">
            <layout class="QGridLayout" name="settings2_frame">
             <property name="leftMargin">
              <number>92</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="1" column="0">
              <widget class="QLabel" name="undo_limit_label">
               <property name="font">
                <font>
                 <family>Ableton Sans Light</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Undo limit</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="SpinBox" name="undo_limit_spinBox">
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="value">
                <number>20</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="recent_limit_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Ableton Sans Light</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Recent files limit</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="SpinBox" name="recent_limit_spinBox">
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="CaptionLabel" name="target_peak_parameter_label">
               <property name="text">
                <string>Target peak parameter</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="ComboBox" name="target_peak_parameter_combo_box"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Page4hidden">
         <attribute name="title">
          <string/>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TabWidget</class>
   <extends>QTabWidget</extends>
   <header location="global">tabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
